### PR TITLE
Governance docs per CE PR 1226

### PR DESCRIPTION
**Pull request by duglin**
See: https://github.com/cloudevents/spec/pull/1226

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [Lance Ball](https://github.com/lance)
+- [Daniel Bevenius](https://github.com/danbev)
+- [Lucas Holmquist](https://github.com/lholmquist)
+- [Fabio Jose](https://github.com/fabiojose)
+- [Helio Frota](https://github.com/helio-frota)
```
Comment (lholmquist) [inline MAINTAINERS.md:9]: I'm pretty sure myself and lance would be the only two as active maintainers at this point

Comment (duglin) [reply MAINTAINERS.md:9]: I was just going by what i see here: https://github.com/orgs/cloudevents/teams/sdk-javascript-maintainers
If you'd like you could start a vote to reduce it down to just you two per: https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md

Comment (lholmquist) [reply MAINTAINERS.md:9]: sure

**Comment by duglin** [comment]
ping @lance @danbev @lholmquist @fabiojose @helio-frota 
We can always do another PR after the maintainer list has been pruned.

**Review by lance — APPROVED** [review]
/lgtm
Agree the maintainers list needs pruning. That can come after this.

**Review by lholmquist — APPROVED** [review]

**Comment by helio-frota** [comment]
> Agree the maintainers list needs pruning. That can come after this.

+1

and also agree with @lholmquist

**Pull request merged**